_docs/setup/install-kubernetes.md

```diff
@@ -14,34 +14,36 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
 
-* The following instructions assume you have access to a Kubernetes cluster with version > 1.7.0
-  and  [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled. To install Kubernetes locally,
-  try [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
+* Depending on your Kubernetes provider:
+  * To install Istio locally, setup [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
 
-* If you are using [Google Container Engine](https://cloud.google.com/container-engine), find out your cluster
+  * If you are using [Google Container Engine](https://cloud.google.com/container-engine), find out your cluster
   name and zone, and fetch credentials for kubectl:
-  
-  ```bash
-  gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <project-name>
-  ```
 
-* If you are using [IBM Bluemix Container Service](https://www.ibm.com/cloud-computing/bluemix/containers), find out your cluster name, and fetch credentials for kubectl:
+    ```bash
+    gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <project-name>
+    ```
 
-  ```bash
-  $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
-  ```
-* If you are using [Openshift Origin](https://www.openshift.org) version 3.7 or later, Openshift by default does not allow containers running with uid 0, you can enable this for Istio's service accounts for ingress and egress as follows:
+  * If you are using [IBM Bluemix Container Service](https://www.ibm.com/cloud-computing/bluemix/containers), find out your cluster name, and fetch credentials for kubectl:
 
-  ```bash
- oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
- oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
-  ```  
+    ```bash
+    $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
+    ```
+  * If you are using [Openshift Origin](https://www.openshift.org) version 3.7 or later, Openshift by default does not allow containers running with uid 0, you can enable this for Istio's service accounts for ingress and egress as follows:
+
+    ```bash
+    oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
+    oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
+    ```  
 
 * Install the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), or upgrade to the latest
-  version supported by your cluster.
+  version supported by your cluster (version 1.7 or newer for CRD support, and it should match your cluster's).
 
-* Upgrading Istio is not yet supported. If you previously installed Istio on this cluster, please uninstall first by following the
+* Note that upgrading Istio is not yet supported. If you previously installed Istio on this cluster, please uninstall first by following the
```

Comment:
Do we have to make such a general statement? Can we say
"If you have a Istio 0.1 cluster installed, please uninstall first by ..".

Reply:
it was there in the previous PR ;-) but I agree with you

Reply:
how about:

If you installed our 0.1 alpha please [uninstall](https://istio.io/v-0.1/docs/tasks/installing-istio.html#uninstalling) it first.

Reply:
hmm it ? them ?

Reply:
nix alpha. The rest is fine.

Reply:
If you have a Istio v0.1.x cluster installed, please uninstall Istio first. ?